extensions/nginx/index.js

```diff
@@ -198,8 +208,12 @@ class NginxExtension extends cli.Extension {
                     confFile,
                     '/etc/nginx/sites-available'
                 ).then(
-                    () => this.ui.sudo(`ln -sf /etc/nginx/sites-available/${confFile} /etc/nginx/sites-enabled/${confFile}`)
-                );
+                    () => {
```

Comment:
quick formatting comment - this can still be a simple arrow function: `() => this.ui.sudo(...)`. If we for sure want to make it a normal arrow function (e.g. `() => { return this.ui.sudo(...); }`) can we move it up like this:

```js
.then(() => {
  return this.ui.sudo();
}).catch()

Reply:
🙈 Oops... Forgot to make it a normal arrow function again. I only changed it to log some stuff for myself...